jt-classbreaks/src/main/java/it/geosolutions/jaiext/classbreaks/NaturalBreaksHistogramOpImage.java

```diff
@@ -166,5 +167,21 @@ protected void postCalculate(Classification c, int band) {
         }
         breaks[0] = buckets.get(0).getMin();
         hc.setBreaks(band, breaks);
+        if (percentages.booleanValue()) {
```

Comment:
This is istantiates a dangling percentagesManager that does not get used?

Reply:
fixed